### ISSUE TITLE
Phone overwrites symlink from Dropbox onto desktop

### ISSUE THREAD

**Issue by JeffreyBenjaminBrown** [DanielDe/org-web#51]
This project looks wonderful.

My first question could be dispatched with a five word sentence in the README: To use this from your phone, you don't need to install an app, just use a browser, correct?

Second, I'm confused about how symlinks work. You write the following in the README:
```
If you don’t already keep your org files in Dropbox or Google Drive, I recommend symlinking them in:

ln -s ~/Documents/todo.org ~/Dropbox/todo.org
```

The syntax for `ln` is such that the target comes first, and the symbolic name for it second. So you seem to be saying I can keep my document on my computer, and just keep a symlink to it on Dropbox.

How can that possibly work? If I edit a file from my phone while my computer is off, Dropbox won't be able to write the changes to the file until the computer is back on. If I have also edited the file from the computer while it was offline, I'll get a merge conflict.

When I make a symlink like you suggested, such that the data is on my desktop and Dropbox contains a link to it, I am indeed able to read the file from my phone. But if I edit the file from my phone, the symlink is replaced by a regular file with the new data -- so now I've got two copies that are out of sync.

**Comment by DanielDe** [comment]
> To use this from your phone, you don't need to install an app, just use a browser, correct?

Yep! This is just a website, no app required. 

> But if I edit the file from my phone, the symlink is replaced by a regular file with the new data -- so now I've got two copies that are out of sync.

Hmm, this doesn't happen for me. In my case when I sync from org-web, the original file is updated on my desktop when Dropbox syncs and I can view it just fine. What desktop OS are you using?

> If I have also edited the file from the computer while it was offline, I'll get a merge conflict.

Yep, this is an issue with using something like Dropbox. In practice I'm just slightly more careful about editing my org file in 2 places like that.

**Comment by JeffreyBenjaminBrown** [comment]
I'm using Linux, specifically Kubuntu 18.04. (Kubuntu = KDE + Ubuntu.) I had serious problems with Dropbox, never got it to work right -- the desktop seemed to behave, but the laptop would open a long text file in Kate and say something about a bad nonce, and after various removals and reinstallations the problem spread to my desktop ... So I gave up; instead I'm just keeping a git repo on a USB stick, cloning it to my laptop and desktop, and foregoing using it on the phone. 

But the experience makes me wonder whether the sync and the editing functions in your app are separable. I'd love to sync my phone somehow to the same repo (maybe by keeping the repo on Dropbox, and using Git for Phones, if such a thing exists), and then use your editor on the phone.

**Comment by DanielDe** [comment]
Ah, unfortunately the only way to view and edit files in org-web currently is to use one of the sync backends. There's lots of interest in other sync backends, including something like Git (see #5), but those are all a ways out. 

Perhaps another thing you can try is reversing the symlink? So put the actual file in Dropbox and then symlink from wherever you're used to accessing it now. Maybe that'd behave better?

**Comment by jonhind** [comment]
I agree about the five word thing - it took me ages to figure it wasn't
necessary to install software.

One way if easily handling symlinks and general file stuff in Linux might
be midnight commander (mc)
